### PR TITLE
Stop deploy-target from auto committing to work with the new deployment flow

### LINKED ISSUE #1305: Deploy entity link takes a long time and auto deploys
If you open https://demo.jujucharms.com/?deploy-target=cs:trusty/wordpress-4 it takes a very long time for the wordpress service to appear on the canvas without any feedback to the user. Then the charm or bundle is deployed which I believe is wrong. Charms and bundles should land uncommitted.

### PR DESCRIPTION
**Pull request by hatched**
Fixes #1305

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Visit the gui with the url suffix `?deploy-target=cs:ceph-268` and it should add the charm to the canvas. You should then be able to commit and work in sandbox as usual.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6264/
Test PASSed.

File: jujugui/static/gui/src/app/views/utils.js [hunk 1]
```diff
@@ -895,42 +904,30 @@ YUI.add('juju-view-utils', function(Y) {
...
     } else {
       // If it's not a bundle then it's a charm.
-      charmstore.getEntity(entityId.replace('cs:', ''), function(error, charm) {
+      charmstore.getEntity(entityId.replace('cs:', ''), (error, charm) => {
         if (error) {
           failureNotification(error);
         } else {
           charm = charm[0];
-          var config = {},
+          let config = {},
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/views/utils.js]: const for objects.

**Comment by hatched** [comment]
Thanks @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**